void/tests/test_math.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+
+from void import math_utils
+
+
+class CalculatePolyTest(unittest.TestCase):
+    def test_calculate_poly(self):
```

Comment:
Let's add a few more test cases just to be sure that edge cases work. We could do something like
```python
values = math_utils.calculate_poly((0, 0), 3, 3, 0)
values = [list(map(lambda x: round(x, 4), x)) for x in values]
expected = [[1.5, -1.5],
            [-1.5, 1.5],
            [1.5, 1.5],
            [-1.5, -1.5]]
``` 

```python
values = math_utils.calculate_poly((0, 0), 3, 3, 90)
values = [list(map(lambda x: round(x, 4), x)) for x in values]
expected = [[1.5, -1.5],
            [-1.5, 1.5],
            [1.5, 1.5],
            [-1.5, -1.5]]
``` 

And one that is not a square:
```python
values = math_utils.calculate_poly((0, 0), 4, 2, 90)
values = [list(map(lambda x: round(x, 4), x)) for x in values]
expected = [[1, -2],
            [-1, 2],
            [1, 2],
            [-1, -2]]
``` 

Reply:
And then to avoid repeating that rounding:

```python
@staticmethod
def round(values):
     return [list(map(lambda x: round(x, 4), x)) for x in values]

def test_something....
    values = math_utils.calc...
    values = self.round(values)
    expected = ...

Reply:
ok